src/app/(space)/(content)/[[...pathname]]/page.tsx

```diff
@@ -114,24 +123,38 @@ export async function generateViewport({ params }: { params: PagePathParams }):
     };
 }
 
-export async function generateMetadata({ params }: { params: PagePathParams }): Promise<Metadata> {
+export async function generateMetadata({
```

Comment:
I think the whole part here it not needed.

Reply:
Nevermind I read the other comment. we should then try to reuse a function in between the 2 to avoid different logic growing over time.

Reply:
Done https://github.com/GitbookIO/gitbook/pull/2376/commits/871f9642adf94d37cae8be3fd50f36df8d2fa542